# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webaudio/AudioNode.cpp
# Commit: d0e8d413cd1c
# Full Hash: d0e8d413cd1c6aa414cf167b085f635ae857e408
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-10-02 16:35:54
# Regressor Bug: 1493613
# File Overlap Count: 1
# Description:
#   Bug 1454998 - Remove track blocking from MediaInputPort. r=karlt
#   
#   This functionality is not used since we moved to only having a single track per
#   MediaStream (bug 1493613).
#   
# ==============================================================================

diff -r 3ce4dc7e9ae2 -r d0e8d413cd1c dom/media/webaudio/AudioNode.cpp
--- a/dom/media/webaudio/AudioNode.cpp	Wed Oct 02 08:12:18 2019 +0000
+++ b/dom/media/webaudio/AudioNode.cpp	Wed Oct 02 08:12:28 2019 +0000
@@ -228,8 +228,7 @@
     MOZ_ASSERT(aInput <= UINT16_MAX, "Unexpected large input port number");
     MOZ_ASSERT(aOutput <= UINT16_MAX, "Unexpected large output port number");
     input->mStreamPort = destinationStream->AllocateInputPort(
-        mStream, AudioNodeStream::AUDIO_TRACK, TRACK_ANY,
-        static_cast<uint16_t>(aInput), static_cast<uint16_t>(aOutput));
+        mStream, static_cast<uint16_t>(aInput), static_cast<uint16_t>(aOutput));
   }
   aDestination.NotifyInputsChanged();
 
@@ -268,8 +267,7 @@
     // Setup our stream as an input to the AudioParam's stream
     MOZ_ASSERT(aOutput <= UINT16_MAX, "Unexpected large output port number");
     input->mStreamPort =
-        ps->AllocateInputPort(mStream, AudioNodeStream::AUDIO_TRACK, TRACK_ANY,
-                              0, static_cast<uint16_t>(aOutput));
+        ps->AllocateInputPort(mStream, 0, static_cast<uint16_t>(aOutput));
   }
 }
 